### PR TITLE
[AC-67: Feature] Create navigation sidebar

### PR DESCRIPTION
**Pull request by turnkevin**
### Provide screenshots of no warning, error in console.

- [x] No merge conflict with the `develop` branch.
- [x] Check PR title is correct, (i.e `[AC-xx: Feature] Create pull request template`).
- [x] Check your app will run without crashes.
- [x] Check if there are hard coded values.
- [x] No `//eslint-disable-next-line no-unused-vars`, unless otherwise specified.
- [x] No unnecessary spaces between lines of code.
- [x] Check you are using `camelCase && PascalCase` correctly.
- [x] Check you named your code properly.
- [x] Remove unused comments.

### CODE REVIEW

**Comment by NorrisWu0** [comment]
Your pr need some fixing first

File: src/common/components/generics/Sidebar.module.scss [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+  width: 30%;
+  border-radius: 50%;
+}
+
+.sidebar ul {
+  position: relative;
+  padding-left: 1rem;
+  display: block;
+}
+.sidebar ul li {
```
Comment (NorrisWu0) [inline src/common/components/generics/Sidebar.module.scss]: You can group this by doing

```scss
.sidebar {
...
  ul {
  ...
 
    li { 
    ...
    }
  }
}
```

File: src/common/components/generics/Sidebar.tsx [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import { BsMegaphone } from 'react-icons/bs';
+import { Image } from '@chakra-ui/react';
+import MenuItem from './MenuItem';
+import styles from './Sidebar.module.scss';
+
+function Sidebar() {
+  const navItems = [
+    {
+      href: '/',
+      content: 'Links',
```
Comment (NorrisWu0) [inline src/common/components/generics/Sidebar.tsx]: Missing localisation support

File: src/common/components/generics/MenuItem.tsx [hunk 3]
```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```
Comment (NorrisWu0) [inline src/common/components/generics/MenuItem.tsx]: You can group these resource under `/generics/Sidebar`

File: src/pages/admin/index.tsx [hunk 4]
```diff
@@ -1,5 +1,10 @@
 import React from 'react';
+import Sidebar from '@src/common/components/generics/Sidebar';
```
Comment (NorrisWu0) [inline src/pages/admin/index.tsx]: Import this component from the index instead

**Review by NorrisWu0 — CHANGES_REQUESTED** [review]

**Comment by NorrisWu0** [comment]
Try to find a solution to fix the failed cypress check as well

File: src/common/components/generics/Sidebar.module.scss (hunk 1 above)
Comment (turnkevin) [reply src/common/components/generics/Sidebar.module.scss]: grouping done.

File: src/common/components/generics/Sidebar.tsx (hunk 2 above)
Comment (turnkevin) [reply src/common/components/generics/Sidebar.tsx]: localisation added.

File: src/common/components/generics/MenuItem.tsx (hunk 3 above)
Comment (turnkevin) [reply src/common/components/generics/MenuItem.tsx]: Grouping finish

File: src/pages/admin/index.tsx (hunk 4 above)
Comment (turnkevin) [reply src/pages/admin/index.tsx]: work done.

**Review by NorrisWu0 — APPROVED** [review]

**Pull request merged**